scout/log/log.py

```diff
@@ -0,0 +1,63 @@
+import os
+import sys
+import logging
+
+def init_log(logger, filename=None, loglevel=None):
```

Comment:
We concluded that it should allways be up to the user how the loggers should be formated.
So if you import the package you set the loggers yourself.